ui/ops/src/traits/airQuality/airQuality.js

```diff
@@ -150,3 +151,27 @@ export function useAirQuality(value) {
     tableData
   };
 }
+
+/**
+ * @param {MaybeRefOrGetter<string|ListAirQualityHistoryRequest.AsObject>} req - The request
+ * @param {MaybeRefOrGetter<boolean>=} paused - Whether to pause the data stream
+ * @return {Promise<ListAirQualityHistoryResponse.AsObject>}
+ */
+export function useListAirQualityHistory(req, paused = false) {
+  const tracker = reactive(
+      /** @type {ActionTracker<AirQualityRecord.AsObject>} */
+      newActionTracker()
+  );
+  return listAirQualitySensorHistory(req, tracker);
+}
```

Comment:
I'm not sure what the point on this function is. It's not reactive in any way and also makes it hard to track loading and error states. The caller should probably just use `src/api/` instead.